Automattic/jetpack
# Boost: Fix Critical CSS resume on module re-enable

Fixes #20975 

#### Changes proposed in this Pull Request:
* Fix issue with Critical CSS not resuming when the feature is being re-enabled
* `hasGenerateRun` variable under `projects/plugins/boost/app/assets/src/js/utils/generate-critical-css.ts` is reset to false once the `critical-css` module is disabled. This lets the Critical CSS generator continue from where it had previously stopped.

#### Jetpack product discussion
NA

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to a site with Jetpack Boost Plugin activated.
* Navigate to `wp-admin/admin.php?page=jetpack-boost`.
* Disable and then enable `Optimize CSS Loading`.
* Critical CSS generation should resume.


https://user-images.githubusercontent.com/8727183/132845000-5129ea94-ba7a-4176-84e3-bb4b9b18c365.mov
